spec/lib/locasms/client_spec.rb

```diff
@@ -25,30 +25,30 @@
     end
 
     it 'Should not send SMS' do
-      subject.should_receive(:numbers)
+      expect(subject).to receive(:numbers)
         .once
         .with([:a, :b, :c])
         .and_raise(LocaSMS::Exception)
 
-      rest_client.should_receive(:get).never
+      expect(rest_client).to receive(:get).never
 
-      lambda{ subject.deliver('given message', :a, :b, :c) }.should raise_error(LocaSMS::Exception)
+      expect{ subject.deliver('given message', :a, :b, :c) }.to raise_error(LocaSMS::Exception)
```

Comment:
Line is too long. [95/80]<br>Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.